asllib/tests/ASLSemanticsReference.t/SemanticsRule.EConcat.asl

```diff
@@ -1,8 +1,8 @@
 func main () => integer
 begin
 
-  let x = [['10','11']];
+  let x = '10' :: '11';
```

Comment:
consider adding concatenation of empty bitvectors:
let y= '' :: '';
assert y == '';